libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -136,6 +136,9 @@ const AP_Scheduler::Task AP_Vehicle::scheduler_tasks[] = {
 #if HAL_GYROFFT_ENABLED
     SCHED_TASK_CLASS(AP_GyroFFT,   &vehicle.gyro_fft,       update,                  400, 50),
     SCHED_TASK_CLASS(AP_GyroFFT,   &vehicle.gyro_fft,       update_parameters,         1, 50),
+#endif
+#ifdef HAVE_AP_BLHELI_SUPPORT
```

Comment:
Should this go right down the bottom of the table?

People were concerned about scheduler ordering - I have a pending PR because things in this table get run before the sub-vehicle's tasks IIRC.

Reply:
I put it at the bottom because it was less critical - do you think it should go somewhere else?